### PR TITLE
Minor update to Rubocop rules

### PR DESCRIPTION
**Pull request by blocknotes**
## Description
- Apply `bundle exec rubocop --auto-gen-config`
- Fix a spec which can't be ignored by RuboCop

## Motivation and Context
Running Rubocop locally shows some issues, we could ignore them for now and take care of them carefully in future.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: spec/controllers/spree/orders_controller_spec.rb [hunk 1]
```diff
@@ -48,8 +48,7 @@
             receive(:valid?).and_return(false)
           )
           allow_any_instance_of(Spree::LineItem).to(
-            receive_message_chain(:errors, :full_messages).
-              and_return(["Order population failed"])
+            receive_message_chain(:errors, :full_messages).and_return(["Order population failed"])
```
Comment (kennyadsl) [inline spec/controllers/spree/orders_controller_spec.rb:51]: why this change?

Comment (blocknotes) [reply spec/controllers/spree/orders_controller_spec.rb:51]: Ouch, I forgot to remove it.
I was evaluating how many issues appear when we clean _rubocop_todo_.
Updating...

Comment (blocknotes) [reply spec/controllers/spree/orders_controller_spec.rb:51]: Correction 😅 
That change is needed, it can be handled by Rubocop auto-correct.

**Review by kennyadsl — APPROVED** [review]
